manifest.json

```diff
@@ -16,7 +16,7 @@
 	},
 	
 	"background": {
-            "scripts": ["notification/data.js","notification/notification.js", "normalView/normalView.js","scripts/background.js"]	
+            "scripts": ["notification/data.js","normalView/normalView.js","notification/notification.js", "scripts/background.js"]	
```

Comment:
The normalView.js should be part of the scripts declared here, is not a background global script, is going to be loaded as part of the normalView template, as you declared in the html with the script tag.